uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -98,17 +98,17 @@ define([
           max: '00:00',
           amount: 0,
           maxAmount: 0,
-          disabled: true,
-          loading: false
+          loading: false,
+          amountExpanded: false
```

Comment:
Lets try to sort these properties in alphabetical order.

Reply:
Done.